lib/useLocomotiveScroll.hook.tsx

```diff
@@ -1,8 +1,18 @@
-import { useContext } from 'react'
+import { useContext, useEffect } from 'react'
 import { LocomotiveScrollContext, LocomotiveScrollContextValue } from './LocomotiveScroll.context'
 
 export function useLocomotiveScroll(): LocomotiveScrollContextValue {
-  return useContext(LocomotiveScrollContext)
+  const context = useContext(LocomotiveScrollContext)
+
+  useEffect(() => {
```

Comment:
This could be fired on the first render with the `scroll` object as `null` before update. I need to try this and modify it if needed before merge